app/Http/Controllers/App/CampaignsController.php

```diff
@@ -69,9 +69,33 @@ public function browse(Request $request)
                 },
             ]
         )
-        ->whereNull('deleted_at')->get()
+            ->whereNull('deleted_at')->get()
         ;
 
+        return self::json(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the code after this "return". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1763)